packages/integration-tests/api/distribution.js

```diff
@@ -55,6 +57,41 @@ async function getDistributionApiFileStream(fileUrl, accessToken) {
   return got.stream(s3SignedUrl);
 }
 
+
+/**
+ * Invoke distribution api lambda directly to get a signed s3 URL.  This is
+ * used in integration testing to we can be sure to use the lambda's iams
+ * permisions to generate correct access.
+ *
+ * @param {string} fileUrl
+ *   Distribution API file URL to request
+ * @param {string} accessToken
+ *   Access token from OAuth provider
+ * @returns {string}
+ *   signed s3 URL for the requested file.
+ */
+async function invokeLambdaForS3SignedUrl(fileUrl, accessToken) {
+  const lambda = new Lambda();
+  const FunctionName = `${process.env.stackName}-ApiDistribution`;
+
+  const path = fileUrl.replace(process.env.DISTRIBUTION_ENDPOINT, '/').replace('//', '/');
```

Comment:
Hmm, this is just because sometimes `fileUrl` is passed to `got` to request the local Express server but here it should just be a relative path on the distribution API? Any way we can distinguish between `filePath` and `fileUrl` in the calling code and only pass in `filePath` here?

Reply:
I was going to say that didn't make sense, but it looks like I'm only getting the fileUrl from a different helper function. So yeah.  I can clean that up probably.  I was thinking we might be able to replace some of the calls to `getDistributionApiResponse` with this call to the lambda in the future though.  (maybe?)

Reply:
Yeah, I think I'd like to probably replace all of them unless there's some reason not to, but nothing occurs to me. Still, I'd like to pass into this function only when it needs instead of having to transform the argument after we receive it

Reply:
I think you'll like what I did there.